packages/main/src/themes/SliderBase.css

```diff
@@ -0,0 +1,124 @@
+:host([disabled]) {
```

Comment:
Each html element has built-in attribute "hidden" and we have to ensure if "hidden" is set, the element will hide. So you have to move the "display: inline-block" in this selector:

```css
:host(:not([hidden])) {
	display: inline-block;
}
```